File: SpearoGo/Models/SolunarData.swift
import Foundation

struct SolunarData {
    let moonPhase: Double         // 0.0 (new) – 1.0 (full cycle)
    let moonIllumination: Double  // 0.0 – 1.0
    let moonrise: Date?
    let moonset: Date?
    let sunrise: Date?
    let sunset: Date?
    let nextMajorPeriod: Date?    // ~2h window of peak activity
    let nextMinorPeriod: Date?    // ~1h window of minor activity
    let activityRating: String    // "Excellent", "Good", "Fair", "Poor"
    let fetchedAt: Date
}

File: wear/app/src/main/java/com/spearotracker/spearogo/models/SolunarData.kt
package com.spearotracker.spearogo.models

data class SolunarData(
    val moonPhase: Double,         // 0.0 (new) - 1.0 (full cycle)
    val moonIllumination: Double,  // 0.0 - 1.0
    val moonrise: Long?,
    val moonset: Long?,
    val sunrise: Long?,
    val sunset: Long?,
    val nextMajorPeriod: Long?,    // ~2h window of peak activity
    val nextMinorPeriod: Long?,    // ~1h window of minor activity
    val activityRating: String,    // "Excellent", "Good", "Fair", "Poor"
    val fetchedAt: Long = System.currentTimeMillis()
)
